substrate/frame/revive/src/exec.rs

```diff
@@ -295,6 +295,9 @@ pub trait Ext: sealing::Sealed {
 	/// Returns a reference to the account id of the current contract.
 	fn account_id(&self) -> &AccountIdOf<Self::T>;
 
+	/// Returns the balance of an AccountId
+	fn account_balance(&self, who: &AccountIdOf<Self::T>) -> U256;
+
```

Comment:
Do we need that on the trait? Long term we should get rid of the T associated type, since all runtime types should not be generic over T but use well-known types (such as H160, U256, ...)

Reply:
Yeah I agree, however I was not entirely sure what to do about for now, as I need the AddressMapper somehow..

Reply:
This function should be removed. Why would you need it to access the `AddressMapper`? The type you are implementing on is generic over `T`.

Reply:
Ah yes of course 😅